apps/studio/src/components/PageEditor/LinkEditorModal.tsx

```diff
@@ -44,31 +42,16 @@ interface PageLinkElementProps {
 
 const PageLinkElement = ({ value, onChange }: PageLinkElementProps) => {
   const { siteId } = useQueryParse(editSiteSchema)
-
-  const selectedResourceId = getResourceIdFromReferenceLink(value)
-
-  const { data: resource } = trpc.resource.getWithFullPermalink.useQuery({
-    resourceId: selectedResourceId,
-  })
-
   return (
-    <>
-      <ResourceSelector
-        siteId={String(siteId)}
-        onChange={(resourceId) =>
-          onChange(getReferenceLink({ siteId: String(siteId), resourceId }))
-        }
-        selectedResourceId={selectedResourceId}
-      />
-
-      {!!resource && (
-        <Box bg="utility.feedback.info-subtle" p="0.75rem" w="full" mt="0.5rem">
-          <Text textStyle="caption-1">
-            You selected /{resource.fullPermalink}
-          </Text>
-        </Box>
-      )}
-    </>
+    <ResourceSelector
+      interactionType="link"
+      siteId={siteId}
+      onChange={(resourceId) =>
+        onChange(getReferenceLink({ siteId: String(siteId), resourceId }))
+      }
+      selectedResourceId={getResourceIdFromReferenceLink(value)}
+      fileExplorerHeight={12}
```

Comment:
nit: the props for `ResourceSelector` might benefit from extending `BoxProps` from chakra so this `fileExplorerHeight` + `hasAdditionalLeftPadding` props can be written naturally